modules/rtm/src/exp_int.cpp

```diff
@@ -0,0 +1,42 @@
+#include <Rcpp.h>
+using namespace Rcpp;
+
+double exp_int(double k){
+    double tau;
+    double xx, yy;
+    if(k <= 0.0){
+        tau = 1;
+    }
+    else if (k > 0.0 and k <= 4.0){
+        xx = 0.5 * k - 1.0;
+        yy=(((((((((((((((-3.60311230482612224e-13
+        *xx+3.46348526554087424e-12)*xx-2.99627399604128973e-11)
```

Comment:
> numerical approximation of the exponential integral

I don't know enough about this particular algorithm. But this sounds like a sufficient rationale (as opposed to, say, these representing some model parameter)